src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java

```diff
@@ -77,19 +77,36 @@ public boolean hasResponseFullAdmit() {
 
     @Override
     public boolean defendantRespondedWithFullAdmitAndPayImmediately() {
+        if (isClaimProceedInCaseMan()) {
+            return false;
+        }
         return hasResponseFullAdmit()
             && isPayImmediately();
     }
 
+    private boolean isClaimProceedInCaseMan() {
```

Comment:
I think you can move this method up to the parent class, together with FeatureToggleService.

Reply:
updated